refocus-bdk-client.js

```diff
@@ -255,7 +144,8 @@ module.exports = (config) => {
     findRoom: (id) => {
       log.debug('Find Room ',
         { id, route: `${SERVER}${API}${ROOMS_ROUTE}/${id}` });
-      return genericGet(`${SERVER}${API}${ROOMS_ROUTE}/${id}`, TOKEN);
+      return generic.get(`${SERVER}${API}${ROOMS_ROUTE}/${id}`,
+        TOKEN, MAX_RETRIES, log);
```

Comment:
Is it correct that MAX_RETRIES is passed in here? Same with the MAX_RETRIES in all the calls below